# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../qemu/qemu-0.9.0-enforce-16byte-boundaries.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

2007-04-12  rediffed for qemu 0.9.0

2006-02-12  Gwenole Beauchesne  <gbeauchesne@mandriva.com>

        * Enforce 16-byte boundaries.

diff -ruN qemu-0.9.0/Makefile.target qemu-0.9.0-enforce-16byte-boundaries/Makefile.target
--- qemu-0.9.0/Makefile.target	2007-02-06 00:01:54.000000000 +0100
+++ qemu-0.9.0-enforce-16byte-boundaries/Makefile.target	2007-04-12 17:29:20.000000000 +0200
@@ -74,7 +74,7 @@
 
 ifeq ($(ARCH),i386)
 HELPER_CFLAGS+=-fomit-frame-pointer
-OP_CFLAGS+=-mpreferred-stack-boundary=2 -fomit-frame-pointer
+OP_CFLAGS+=-fomit-frame-pointer
 ifeq ($(HAVE_GCC3_OPTIONS),yes)
 OP_CFLAGS+= -falign-functions=0 -fno-gcse
 else
